docs/convenience_commands.md

```diff
@@ -0,0 +1,126 @@
+# Convenience commands
+
+In addition to the primary commands for the build, imputation, and
+resequencing pipelines, PHGv2 also provides a suite of "convenience
+commands" for miscellaneous "quality of life (QoL)" improvements. In 
+this document, we will discuss the currently available external
+commands for performing highly used tasks.
+
+
+## Convert gVCF files to hVCF files
+
+> Create hVCF files from existing gVCF files created by the PHG
+
+**Command** - `gvcf2hvcf`
+
+**Example**
+
+``` shell
+phg gvcf2hvcf \
+    --bed my/bed/file.bed \
+    --reference-file my/updated/ref/fasta.fa \
+    --gvcf-dir gvcf/directory \
+    --db-path my/phg/db
+```
+
+**Parameters**
```

Comment:
I'm going to add conda-env-prefix as we do access AGC with ProcessBuilder() calls, so this needs to be here.  Note that hvcf files are written to the gvcf-dir location.  Perhaps that should be mentioned.